src/test/java/org/isf/ward/test/Tests.java

```diff
@@ -169,7 +169,7 @@ public void testIoGetOpdWards() throws Exception {
 		List<Ward> wards = wardIoOperation.getOpdWards();
 		
 		// then:
-		assertThat(wards.size() == 0);
+		assertThat(wards).hasSize(1);
```

Comment:
hasSize(0)

Reply:
No; the previous test had no condition so it didn't realize that `0` was wrong.   Notice all the tests passed so `1` is the correct value.